### PR TITLE
Restore the mender-connect dependency and re-enable test

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: recipes/mender-connect/debian-master/control [hunk 1]
```diff
@@ -7,7 +7,7 @@ Build-Depends: debhelper (>= 9)
 
 Package: mender-connect
 Architecture: any
-Depends: mender-client (>3.1.99), ${shlibs:Depends}, ${misc:Depends}
+Depends: mender-client (>3.2.0), ${shlibs:Depends}, ${misc:Depends}
```
Comment (lluiscampos) [inline recipes/mender-connect/debian-master/control:10]: This is not strictly speaking necessary, but a nice clean-up IMO.

**Review by lluiscampos — CHANGES_REQUESTED** [review]
[This fixture](https://github.com/mendersoftware/mender-dist-packages/blob/master/tests/conftest.py#L97-L101) is what you need to update for the addon tests to install a given version of the client.

**Comment by lluiscampos** [comment]
Actually the JIRA task was a bit off in the description (because the original PR got force-pushed), but the original commit to revert is [this one](https://github.com/mendersoftware/mender-dist-packages/pull/147/commits/42e4e97291d7f5929f32b3eb23dc98caa059f160).

**Comment by oleorhagen** [comment]
> Actually the JIRA task was a bit off in the description (because the original PR got force-pushed), but the original commit to revert is [this one](https://github.com/mendersoftware/mender-dist-packages/pull/147/commits/42e4e97291d7f5929f32b3eb23dc98caa059f160).

:+1: 

  https://github.com/mendersoftware/mender-docs/pull/1655

**Review by lluiscampos — APPROVED** [review]

**Comment by lluiscampos** [comment]
Looks good!

I am sorry I have introduced a merge conflict from another PR :smile_cat:

**Comment by lluiscampos** [comment]
:thinking: I think you need to cherry-pick this one:
* https://github.com/mendersoftware/mender-dist-packages/pull/190

Because the new packages have now this `+debian+buster` prefix.

**Comment by oleorhagen** [comment]
> thinking I think you need to cherry-pick this one:
> 
>     * [Changelog:All: Bump tests/mender_test_containers from `87e4fe7` to `5358b6c` #190](https://github.com/mendersoftware/mender-dist-packages/pull/190)
> 
> 
> Because the new packages have now this `+debian+buster` prefix.

Ahh, nice. Fix is already in. I was planning on looking at this today :raised_hands:

**Pull request merged**